backend/src/main/java/com/bakdata/conquery/models/externalservice/ResultType.java

```diff
@@ -226,6 +227,20 @@ public static class IdT extends PrimitiveResultType {
 		public static final IdT INSTANCE = new IdT();
 	}
 
+	@CPSType(id = "SECONDARY_ID", base = ResultType.class)
+	@NoArgsConstructor(access = AccessLevel.PRIVATE)
+	public static class SecondaryIdT extends PrimitiveResultType {
+		@Getter(onMethod_ = @JsonCreator)
+		public static final SecondaryIdT INSTANCE = new SecondaryIdT();
+	}
+
+	@CPSType(id = "PRIMARY_COLUMN", base = ResultType.class)
+	@NoArgsConstructor(access = AccessLevel.PRIVATE)
+	public static class PrimaryColumn extends PrimitiveResultType {
+		@Getter(onMethod_ = @JsonCreator)
+		public static final PrimaryColumn INSTANCE = new PrimaryColumn();
+	}
```

Comment:
PrimaryColumn verstehe ich gerade nicht. Ist es nicht besser ein Spalte zu haben die einem sagt, was welches die primary column ist oder dass man das in der columnDescription erstmal flagged?

Reply:
So ist es doch in der columnDescription geflagged

Reply:
> vllt. ist es immer String

Es muss String sein, es wird ja ein Konzept drauf aufgebaut

Reply:
ja stimmt, so lange wir da kein typing haben bringt das nicht viel